app/helpers/episodes_helper.rb

```diff
@@ -52,16 +52,13 @@ def episode_status_class(episode)
   end
 
   def episode_media_status(episode)
-    if episode_all_media(episode).any? { |m| upload_problem?(m) }
```

Comment:
Looks like `upload_problem?` is unused now.

Reply:
deleted!